alicorn-expressions.lua

```diff
@@ -1109,35 +1110,27 @@ local block = metalanguage.reducer(
 	"block"
 )
 
--- example usage of primitive_applicative
```

Comment:
this example was deleted, is it no longer applicable?

Reply:
thought they were unused, turns out some of them are still hanging on and apparently not broken